composer/plan.sh

```diff
@@ -20,7 +20,8 @@ do_build() {
 }
 
 do_check() {
-  return 0 # makes no sense here…
+  attach
```

Comment:
Looks like this is probably left over from debugging?

Reply:
oops. Fixed.

Reply:
Thanks @stevendanna